pkg/cluster/storage.go

```diff
@@ -14,6 +15,23 @@ import (
 // This is used when bootstrapping a cluster from a managed database or external etcd cluster.
 // This is NOT used with embedded etcd, which bootstraps over HTTP.
 func (c *Cluster) save(ctx context.Context) error {
+	// check if etcd is still learner
+	localEndpoint := "https://127.0.0.1:2379"
```

Comment:
Can this information be derived from the config on the Cluster object?

Reply:
what would it be exactly, the function needs an endpoint plus the runtime config for the certs

Reply:
No idea. I'm just curious if that info is in the config. If not, no big deal. And if it's not, we can make this a constant.

Reply:
None of this is safe to do on anything except for embedded etcd anyways. Just handle the error message in the Create block down below.